static/js/components/VideoPlayer.js

```diff
@@ -242,11 +243,35 @@ export default class VideoPlayer extends React.Component<*, void> {
     })
   }
 
+  selectPlaylist = () => {
+    const sortByBandwidth = R.sortBy(R.path(["attributes", "BANDWIDTH"]))
+    const playlists = sortByBandwidth(
+      this.player.tech_.hls.playlists.master.playlists
+    )
+    // Always start with highest bandwidth for first 10 seconds
+    if (this.player.tech_.currentTime() < 10) {
+      return _.last(playlists)
+    }
+    // Return playlist with highest bandwidth <= system bandwidth
+    return _.last(
+      R.filter(
+        rep =>
+          rep.attributes.BANDWIDTH <=
+          _.max([
```

Comment:
This is to make sure at least one of the bandwidth options will always be in this list?

Reply:
Yes